opentelemetry-tracing/opentelemetry-dotnet-manual/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenTelemetry .NET manual instrumentation examples
+
+This folder contains sample applications that were manually instrumented using
+[OpenTelemetry .NET](https://opentelemetry.io/docs/instrumentation/net/).
+
+For more information on OpenTelemetry .NET and some of the available instrumentations, see:
+
+- [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet).
+- [OpenTelemetry .NET Contrib](https://github.com/open-telemetry/opentelemetry-dotnet-contrib).
+
+## Libraries and Framework Examples
+
+- [ASP.NET Core, MongoDB, and HttpClient](./aspnetcore-and-mongodb/README.md)
```

Comment:
```suggestion
- [ASP.NET Core, MongoDB and HttpClient](./aspnetcore-and-mongodb/README.md)
```

Reply:
The current way is correct in American English.

https://getproofed.com/writing-tips/differences-american-british-punctuation/